Dockerfile.test.db

```diff
@@ -33,4 +33,6 @@ RUN echo "Use LorisTest;" | cat - \
 
 RUN echo "Use LorisTest;" >> /docker-entrypoint-initdb.d/0001-paths.sql
 RUN echo "UPDATE Config SET Value='${BASE_DIR}/' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='base');" >> /docker-entrypoint-initdb.d/0001-paths.sql
+RUN echo "UPDATE Config SET Value='This is a new hidden S3cret!?' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='JWTKey');" >> /docker-entrypoint-initdb.d/0001-paths.sql
```

Comment:
This should be set in the test setUp function (preferrably to a random string and not a hardcoded one) and restored to the original value in the tearDown, not hardcoded in the Dockerfile, since it's supposed to be a secret key.